src/highlighting.c

```diff
@@ -1776,3 +1776,23 @@ gboolean highlighting_is_code_style(gint lexer, gint style)
 	return !(highlighting_is_comment_style(lexer, style) ||
 		highlighting_is_string_style(lexer, style));
 }
+
+
+gchar **highlighting_get_keywords(GeanyFiletypeID filetype_id)
+{
+	GString *str = g_string_sized_new(1000);
+	gchar **keywords;
+	gchar **keyword_str;
+
+	foreach_strv(keyword_str, style_sets[filetype_id].keywords)
```

Comment:
The use of the macro within the macro doesn't even save a line, it saves about 11 chars by my calculation :(